mpv.py

```diff
@@ -1814,6 +1842,10 @@ def option_info(self, name):
         except AttributeError:
             return None
 
+    def __register_async_callback(self, key: int, callback):
```

Comment:
Elsewhere in the file we use a single underscore for non-public methods. I think we should keep this consistent.

Reply:
:+1: Solved in latest push.